utils/hct/query.py

```diff
@@ -0,0 +1,27 @@
+# user defined function
+def query(insts, inst):
```

Comment:
Change the function name to inst_query?
We may want to add things like pass_query in the future.